src/css/math.less

```diff
@@ -298,6 +298,21 @@
     }
   }
 
+  .mq-limit {
+    text-align: center;
+
+    span  {
+      display: block;
```

Comment:
this is causing every `span` in the limit's "approaches" block to be on its own line: 
![image](https://cloud.githubusercontent.com/assets/225809/13593532/db55b876-e4b0-11e5-9276-0f450453344f.png)